docs/plugins/mapper-annotated-text.asciidoc

```diff
@@ -153,13 +254,13 @@ the equals signs so wil actively reject documents that contain this today.
 Annotations are normally a way of weaving structured information into unstructured text for
 higher-precision search.
 
-`Entity resolution` is a form of document enrichment undertaken by specialist software or people 
+`Entity resolution` is a form of document enrichment undertaken by specialist software or people
```

Comment:
Looks like these changes here are just removing trailing white spaces?

Reply:
Probably a setting of the editor hat applies them when a file is saved.

Reply:
Yes, it's automatic. I could revert it but i think it's probably fine?

Reply:
I think it is fine. It just is a bit confusing when reviewing.